jdi-light-vuetify-tests/src/main/java/io/github/com/custom/OverlayWithCloseButton.java

```diff
@@ -0,0 +1,16 @@
+package io.github.com.custom;
+
+import com.epam.jdi.light.vuetify.elements.common.Overlay;
+import com.epam.jdi.light.vuetify.elements.common.VuetifyButton;
+
+import static com.jdiai.tools.Timer.waitCondition;
+
+public class OverlayWithCloseButton extends Overlay {
+    private static final String CLOSE_BUTTON_PATH = ".v-overlay__content button";
```

Comment:
We do not need .v-overlay__content in locator, as we need to customize the content

Reply:
Changed locator for button.